clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1761,7 +1761,8 @@ public OffsetAndMetadata committed(TopicPartition partition, final Duration time
                     Collections.singleton(partition), time.timer(timeout));
             if (offsets == null) {
                 throw new TimeoutException("Timeout of " + timeout.toMillis() + "ms expired before the last " +
-                        "committed offset for partition " + partition + " could be determined");
+                        "committed offset for partition " + partition + " could be determined. User could tune default.api.timeout.ms " +
```

Comment:
The user is the one reading this message. What do you think of something like "You can tune..."? Or "Try tuning..."?